awx/ui_next/src/screens/Template/shared/LabelSelect.jsx

```diff
@@ -57,13 +57,26 @@ function LabelSelect({ value, placeholder, onChange, onError }) {
     <Select
       variant={SelectVariant.typeaheadMulti}
       onToggle={toggleExpanded}
-      onSelect={onSelect}
+      onSelect={(e, item) => {
```

Comment:
When the label gets to the custom hook `useSyncedSelectedValue` it expects the label to be an object.